proto/sequencerblockapis/astria/sequencerblock/v1alpha1/block.proto

```diff
@@ -86,6 +86,11 @@ message Deposit {
   // the address on the destination chain which
   // will receive the bridged funds
   string destination_chain_address = 5;
+  // the transaction ID of the source action for the deposit, consisting
+  // of the transaction hash.
+  astria.primitive.v1.TransactionId id_of_source_transaction = 6;
```

Comment:
nit: `source_transaction_id`  would match naming conventions more. Generally perfer not to have "of" in names